docs/tutorials/start-a-private-network/4customchain.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Creating Your Private Network
+---
+Now that each participant has their own keys generated, you're ready to start your own custom chain.
+
+> Validators should not share the same keys, even for learning purposes. If two validators have the same keys, they will produce conflicting blocks, and be slashed.
+
+## Create a Chainspec
+
+Last time around, we used `--chain=local` which is a predefined "chainspec" that has Alice and Bob specified as validators along with many other useful defaults.
+
+Rather than writing our chainspec completely from scratch, we'll just make a few modifications to the the one we used before. To start we need to export the chainspec to a json file. Remember, further details about all of these commands are available by running `node-template --help`.
+
+```bash
+./target/release/node-template build-spec --chain=local > customSpec.json
+```
+
+The file we just created contains several fields, and one can learn a lot by exploring them. By far the largest field is a single hex number that is the Wasm binary of our runtime. It is part of what you built earlier when you ran the `cargo build` command. Having this information on-chain enables forkless upgrades.
```

Comment:
```suggestion
The file we just created contains several fields, and you can learn a lot by exploring them. By far the largest field is a single hex number that is the Wasm binary of our runtime. It is part of what you built earlier when you ran the `cargo build` command. Having this information on-chain enables forkless upgrades.
```

Reply:
one -> you is fine with me. Why do you prefer to remove the bit about forkless upgrades?

Reply:
It is just really out of context. I dont remember you really making mention of it anywhere else, and then you drop it pretty casually like the reader knows what that means.

You def can elaborate on it, or better, just keep your tutorial concise about what you are trying to teach, and expand on forkless upgrades as a "learn more" step